go/private/toolchain.bzl

```diff
@@ -36,10 +36,13 @@ def _go_sdk_impl(ctx):
     
   # Build the standard library for valid cross compile platforms
   #TODO: fix standard library cross compilation
+  tmp = ctx.os.environ['TMP']
```

Comment:
Not sure if this is the cause of the Jenkins failure. Do you need to use `dict.get` here if it's not set?